src/app/components/organisms/cart_items/index.tsx

```diff
@@ -194,8 +194,8 @@ const notify = () => toast('product Removed',  {
                 <Separator className='relative mb-14 m-auto w-[1200px]'/>
                 <div className=''>
                   {
-                    cartProducts && cartProducts.map((item, i)=> {
-                      return <div className='pb-7 box-border' key={i}>
+                    cartProducts && cartProducts.map((item)=> {
+                      return <div className='pb-7 box-border'>
                         <CartContent item= {item} key={item.id} />
```

Comment:
You can remove the key prop here, you only need to add it to its first parent component in a loop